package.json

```diff
@@ -19,6 +19,7 @@
         "@dhis2/d2-ui-rich-text": "^7.4.0",
         "@dhis2/d2-ui-sharing-dialog": "^7.3.3",
         "@dhis2/ui": "^9.10.1",
+        "@dhis2-ui/calendar": "9.9.0-alpha.6",
```

Comment:
Alphas releases are useful when developing, but IMHO it is best to wait for the stable `@dhis2/ui` release before merging this PR. Do you know the timeline/steps to get the stable release to include the alpha changes?

Reply:
I was given the task to update the alpha with latest master. But I am not sure about the timeline as there are still accessibility task for CalendarInput.